benchmark/accuracy/score_predictions.jl

```diff
@@ -61,5 +61,9 @@ if haskey(parsed_args, "write-prediction-csv")
     delete!(long_df, :index)
     delete!(long_df, :variable)
     final_df = unstack(long_df, :index_var, :source, :value)
+    final_df[:, :source_id] = [parse(Int, match(r"^\S+", x).match) for x in final_df[:, :index_var]]
+    final_df[:, :property] = [match(r"[a-z].*", x).match for x in final_df[:, :index_var]]
+    delete!(final_df, :index_var)
```

Comment:
This PR looks good. I'm curious what the logic is with this whole section of code that melts the data frame and then unmelts it again. Just a few comment lines would be helpful for the future, especially with the regex matching.

Reply:
Done